vncviewer/GestureHandler.h

```diff
@@ -0,0 +1,97 @@
+/* Copyright 2019 Aaron Sowry for Cendio AB
+ * 
+ * This is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ * 
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this software; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307,
+ * USA.
+ */
+
+#ifndef __GESTUREHANDLER_H__
+#define __GESTUREHANDLER_H__
+
+#include <map>
+
+#include <X11/Xlib.h>
+#include <X11/extensions/XInput2.h>
+
+#include <rfb/Timer.h>
+
+// Internal state bitmasks
+#define GH_NOGESTURE   0
+#define GH_LEFTBTN     1
+#define GH_MIDDLEBTN   2
+#define GH_RIGHTBTN    4
+#define GH_VSCROLL     8
+#define GH_HSCROLL     16
+#define GH_ZOOM        32
```

Comment:
Perhaps bikeshedding, but I would prefer "PAN" and "PINCH" or something like that. It more properly describes the gesture, rather than the intended action (which this code has no say over).

Reply:
My suggestions for renaming the gesture definitions can be found in 10d5a363. If you agree then feel free to resolve this conversation.

Reply:
Looks good. :+1: